tests/bluetooth/tester/testcase.yaml

```diff
@@ -1,5 +1,28 @@
 tests:
   bluetooth.general.tester:
+    build_only: true
+    platform_allow:
+      - qemu_x86
+      - nrf52840dk_nrf52840
+    tags: bluetooth
+    harness: bluetooth
+  bluetooth.general.tester_le_audio:
+    build_only: true
+    platform_allow:
+      - qemu_x86
+      - nrf5340dk_nrf5340_cpuapp
```

Comment:
Why the difference in allowed platforms for each configuration?

Reply:
As there are overlays for nrf5340dk_nrf5340_cpuapp, nrf52840dk_nrf52840, I wanted to include those in build verification system as well. As I know, the nrf5340dk_nrf5340_cpuapp is used for LeAudio while nrf52840dk_nrf52840 for Host weekly runs I just added those platforms.